src/nunit.analyzers/Constants/ConstActualValueUsageAnalyzerConstants.cs

```diff
@@ -0,0 +1,9 @@
+namespace NUnit.Analyzers.Constants
+{
+    internal static class ConstActualValueUsageAnalyzerConstants
+    {
+        public const string Title = "Actual value should not be constant";
+        public const string Message = "Actual value should not be constant - " +
+            "possible that actual and expected values are switched places";
```

Comment:
I'm not a native English speaker, but I think the sentence sounds a bit odd.
```suggestion
            "perhaps the actual and expected values have switched places";
```

Reply:
fixed